docs/complex_parameters.md

```diff
@@ -200,6 +200,6 @@ importKey(Tezos, secretKey)
     println(`Waiting for ${op.hash} to be confirmed...`);
     return op.confirmation(1).then(() => op.hash);
 }).then(hash => {
-    println(`Operation injected: https://better-call.dev/kathmandunet/KT1Resnq6SvWRUXA9FaNczhJ278QzwPjWcGz/operations`);
+    println(`Operation injected: https://better-call.dev/ghostnet/KT1Resnq6SvWRUXA9FaNczhJ278QzwPjWcGz/operations`);
```

Comment:
This operation doesn't exist on Ghostnet. Would it be better to remove these hardcoded links?

Reply:
We need the link to show the user it works. The operation doesn't;t exist yet, as I haven't finished the updates. PR is still a draft, I will get to this one.